pid_tracker.py

```diff
@@ -44,10 +51,21 @@ def configure_callback(self, conf):
     """called by collectd to configure the plugin. This is called only once"""
     for node in conf.children:
       if node.key == 'PidFile':
-        if len(node.children) != 1:
+        if len(node.children) == 0:
           self.collectd.warning('pid-tracker plugin: PidFile missing or too many children: "%s' % node)
+        elif len(node.children) > 2:
+          self.collectd.warning('pid-tracker plugin: PidFile has too many children: "%s' % node)
         else:
-          self.add_pidfile(node.values[0], node.children[0].values[0])
+          plugin_instance = None
+          collect_mem_stats = False
+          for child_node in node.children:
+            if child_node.key == 'PluginInstance':
+              plugin_instance = child_node.values[0]
+            elif child_node.key == 'CollectMemStats':
+              collect_mem_stats = True if child_node.values[0].__str__() in ['True', 'true'] else False
```

Comment:
The distutils method will throw a ValueError which is different than the way I've got things going. If we want the ValueError thrown I'll swap in the distutils method, otherwise I can create a method to use to DRY that up.